apps/server/src/modules/oauth/error/user-not-found-in-unprovisioned-school.loggable-exception.ts

```diff
@@ -0,0 +1,19 @@
+import { ErrorLogMessage, Loggable, LogMessage, ValidationErrorLogMessage } from '@src/core/logger';
+import { OAuthSSOError } from './oauth-sso.error';
+
+export class UserNotFoundInUnprovisionedSchoolLoggableException extends OAuthSSOError implements Loggable {
+	constructor(private readonly externalUserId: string, private readonly systemId: string) {
+		super('Provisioning of user with externalId: externalUserId failed', 'sso_user_notfound_in_unprovisioned_school');
+	}
+
+	getLogMessage(): LogMessage | ErrorLogMessage | ValidationErrorLogMessage {
+		return {
+			message: 'The user could not be found because the school was not provisioned.',
```

Comment:
Now we have 2 different messages. one here and the other one in the constructor/on the OAuthSSOError.
```suggestion
			message: this.message,
```